zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -1393,7 +1393,13 @@ private void completion(NotebookSocket conn, HashSet<String> userAndRoles, Noteb
     }
 
     final Note note = notebook.getNote(getOpenNoteId(conn));
-    List<InterpreterCompletion> candidates = note.completion(paragraphId, buffer, cursor);
```

Comment:
this used to throw exception, I think its a better idea to log that error and send empty array to frontend,